NOTES.ANDROID

```diff
@@ -0,0 +1,66 @@
+
+ NOTES FOR THE ANDROID PLATFORMS
+ ===============================
+
+ Requirement details
+ -------------------
+
+ Beside basic tools like perl and make you'll need to download Android
+ NDK. It's available for Linux, MacOS X and Windows, but only Linux
+ version was actually tested. There is no reason to believe that MacOS X
+ wouldn't work. And as for Windows it's unclear which "shell" would work
+ [best], MSYS2 might have best chances. NDK version should play lesser
+ role, the goal is to support a range of most recent versions.
```

Comment:
How about adding some details for what versions are tested? NDK changes a lot in recent versions (r13~r16), and I think explicit statements are better than "a range of most recent versions"

Reply:
I'd argue in favour of keeping it as generic as possible. Doing otherwise kind of puts pressure on keeping it up-to-date by following NDK releases closely. In other words I'd rather leave it unsaid, and have user fill the gaps...